src/commands/simbrief.ts

```diff
@@ -0,0 +1,13 @@
+import { CommandDefinition } from '../lib/command';
+import { CommandCategory } from '../constants';
+import { makeEmbed } from '../lib/embed';
+
+export const simbrief: CommandDefinition = {
+    name: 'simbrief',
+    description: 'Provides a link to the FlyByWire Simbrief integration tutorial video.',
+    category: CommandCategory.FBW,
+    executor: (msg) => msg.channel.send(makeEmbed({
+        title: 'FlyByWire A32NX | Simbrief Integration',
+        description: 'Watch a tutorial on how to use the FlyByWire A32NX Simbrief Integration [here](https://youtu.be/l_5J7ZuEEtk)',
```

Comment:
I suggest adding an explanation of what this actually means for less experienced users who would typically be the target audience for such command, as well as correctly capitalising the SimBrief name. For example:

```suggestion
        description: 'Please watch our video tutorial on how to use the FlyByWire A32NX SimBrief Integration [here.](https://youtu.be/l_5J7ZuEEtk) This functionality will allow you to import your SimBrief flight plan, including the route, required fuel, passengers and their luggage.',
```